components/jenkins/data.tf

```
@@ -4,10 +4,35 @@ data "azurerm_resource_group" "disks_resource_group" {
   name = "disks-${var.env}-rg"
 }
 
+data "azuread_service_principal" "service_principal" {
+  display_name = "dcd_sp_aks_dcd-cftapps-sbox_v2"
+}
+
+data "azuread_groups" "dts_platform_operations" {
+  display_names = "DTS Platform Operations"
+}
+
+data "azuread_groups" "subidcontributer" {
```

```suggestion
data "azuread_group" "subidcontributer" {
```

Likely my fault for dodgy link sorry - https://registry.terraform.io/providers/hashicorp/azuread/latest/docs/data-sources/group - singular 😁